.github/workflows/generator_container_slsa3.yml

```diff
@@ -48,6 +48,11 @@ on:
         required: false
         type: boolean
         default: false
+      token:
```

Comment:
for secrets inputs to re-usable workflow, there is a separate secret field, see https://github.com/slsa-framework/slsa-github-generator/blob/cd7ae8d43f0389bd7ba47374dd8c93c1219cb589/.github/workflows/generator_container_slsa3.yml#L24

I think this parameter is not needed, in the sense that the default token is already available by default. (we use it to retrieve an OIDC token.)
I think all we need is the change to generate-builder/action.yml: add the `token` input and set the default as `default: ${{ github.token }}`.

Reply:
The `token` input to the `generate-builder` action already defaults to `github.token`. So it should work ok already.

https://github.com/slsa-framework/slsa-github-generator/blob/fc6500582079e2a1c41f181e7f04189b2a94816f/.github/actions/generate-builder/action.yml#L23-L26

I also don't think this might not be necessary. The reusable workflows declare what permissions are needed so the default `github.token` should always have the correct permissions. We also need to use the default token for OIDC to link the provenance to the workflow's identity. Using one token for OIDC and one for checkout and other actions might be a bit confusing to users.